proof/crefine/AARCH64/Refine_C.thy

```diff
@@ -876,18 +876,17 @@ lemma threadSet_all_invs_triv':
   unfolding all_invs'_def
   apply (rule hoare_pre)
    apply (rule wp_from_corres_unit)
-     apply (rule threadset_corresT [where f="tcb_arch_update (arch_tcb_context_set f)"])
+      apply (rule threadset_corresT [where f="tcb_arch_update (arch_tcb_context_set f)"]; simp?)
```

Comment:
given all resulting subgoals get `simp` anyway, and indent for subsequent apply hasn't changed, what does the `simp?` give you?

Reply:
There are new, relatively trivial subgoals that are solved by that simp.